opentelemetry-sdk/src/trace/span_processor.rs

```diff
@@ -134,7 +134,10 @@ impl SpanProcessor for SimpleSpanProcessor {
             .and_then(|mut exporter| futures_executor::block_on(exporter.export(vec![span])));
 
         if let Err(err) = result {
-            global::handle_error(err);
+            otel_error!(
+                name: "SimpleProcessor.OnEnd.Error",
+                reason = format!("{:?}", err)
```

Comment:
do we need error severity? would the information here be duplicate of what an exporter might already be logging?
I am also unsure if the message is really actionable by end users. there is no indication of what happened or what is the impact of this..

Reply:
> would the information here be duplicate of what an exporter might already be logging?

Not necessarily - exporter may choose to return the export status to be logged here. The status may be actionable in that case. For logs, we check the error type and then decide whether to log as `error` or `debug`. Similar logic needs to be done here. Will add a TODO here. Meanwhile, do you want to change this to `debug` ?

Reply:
Added TODO, and changed to otel_debug.